destroy.bash

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+cd tf_files
+LOGIN_NODE=`grep -A20 "aws_eip.login" terraform.tfstate | grep "public_ip" | head -1 | sed 's/[ \",]//g' | cut -d: -f2`
+
+echo "Working with Login Node: $LOGIN_NODE"
+
```

Comment:
https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws-destroy.html the notes seem to suggest no?